xcm/xcm-builder/src/tests/barriers.rs

```diff
@@ -47,7 +47,10 @@ fn computed_origin_should_work() {
 		DescendOrigin(Parachain(100).into()),
 		DescendOrigin(PalletInstance(69).into()),
 		WithdrawAsset((Parent, 100).into()),
-		BuyExecution { fees: (Parent, 100).into(), weight_limit: Limited(100) },
+		BuyExecution {
+			fees: (Parent, 100).into(),
+			weight_limit: Limited(Weight::from_ref_time(100)),
```

Comment:
I would add a new helper function `fn from_dummy(all_fields: u64)` which sets both fields to be equal to `all_fields`, and use that everywhere in these tests.